client/my-sites/email/email-providers-in-depth-comparison/index.tsx

```diff
@@ -1,15 +1,14 @@
 import { useTranslate } from 'i18n-calypso';
-import { FunctionComponent } from 'react';
 import ComparisonTable from 'calypso/my-sites/email/email-providers-in-depth-comparison/comparison-table';
 import {
 	professionalEmailFeatures,
 	googleWorkspaceFeatures,
 } from 'calypso/my-sites/email/email-providers-in-depth-comparison/data';
-import type { EmailProvidersInDepthComparisonProps } from 'calypso/my-sites/email/email-providers-in-depth-comparison/types';
+import type { ReactElement } from 'react';
 
 import './style.scss';
 
-const EmailProvidersInDepthComparison: FunctionComponent< EmailProvidersInDepthComparisonProps > = () => {
+const EmailProvidersInDepthComparison = (): ReactElement => {
```

Comment:
I suspect there is no way to specify the type for the props until we actually pass one to this component?

```suggestion
const EmailProvidersInDepthComparison = ( {}: EmailProvidersInDepthComparisonProps ): ReactElement => {
```

Reply:
I've added a comment to note this limitation. Unfortunately, the linter complains if I use your suggestion (aka an empty destructure) or if I simply specify `props` and then avoid using the props variable in the code.